helpers/scraper.js

```diff
@@ -0,0 +1,366 @@
+  ) {
+    return false;
+  }
+  return true;
+};
+
+const getAbbrev = function(state) {
+  let states = [
+        ['Arizona', 'AZ'],
+        ['Alabama', 'AL'],
+        ['Alaska', 'AK'],
+        ['Arizona', 'AZ'],
+        ['Arkansas', 'AR'],
+        ['California', 'CA'],
+        ['Colorado', 'CO'],
+        ['Connecticut', 'CT'],
+        ['Delaware', 'DE'],
+        ['Florida', 'FL'],
+        ['Georgia', 'GA'],
+        ['Hawaii', 'HI'],
+        ['Idaho', 'ID'],
+        ['Illinois', 'IL'],
+        ['Indiana', 'IN'],
+        ['Iowa', 'IA'],
+        ['Kansas', 'KS'],
+        ['Kentucky', 'KY'],
+        ['Kentucky', 'KY'],
+        ['Louisiana', 'LA'],
+        ['Maine', 'ME'],
+        ['Maryland', 'MD'],
+        ['Massachusetts', 'MA'],
+        ['Michigan', 'MI'],
+        ['Minnesota', 'MN'],
+        ['Mississippi', 'MS'],
+        ['Missouri', 'MO'],
+        ['Montana', 'MT'],
+        ['Nebraska', 'NE'],
+        ['Nevada', 'NV'],
+        ['New Hampshire', 'NH'],
+        ['New Jersey', 'NJ'],
+        ['New Mexico', 'NM'],
+        ['New York', 'NY'],
+        ['North Carolina', 'NC'],
+        ['North Dakota', 'ND'],
+        ['Ohio', 'OH'],
+        ['Oklahoma', 'OK'],
+        ['Oregon', 'OR'],
+        ['Pennsylvania', 'PA'],
+        ['Rhode Island', 'RI'],
+        ['South Carolina', 'SC'],
+        ['South Dakota', 'SD'],
+        ['Tennessee', 'TN'],
+        ['Texas', 'TX'],
+        ['Utah', 'UT'],
+        ['Vermont', 'VT'],
+        ['Virginia', 'VA'],
+        ['Washington', 'WA'], 
+        ['District of Columbia', 'DC'],
+        ['West Virginia', 'WV'],
+        ['Wisconsin', 'WI'],
+        ['Wyoming', 'WY'],
+    ];
+
+    for (var i = 0; i < states.length; i++) {
+      if (state == states[i][0]) {
+        return states[i][1];
+      }
+    }
+    return "";
+}
+
+const getCalifornia = function(school) {
+  let splitSchool = school.split(" ");
+  let usableWords = [];
+  for (var i = 0; i < splitSchool.length; i++) {
+      if (splitSchool[i].includes("!")) {
+        break;
+      } else {
+        usableWords.push(splitSchool[i]);
+      }
+    } 
+    if (school.includes("Los Angeles")) {
+      return "UCLA";
+    }
+  if (school.includes("Berkeley")) {
+    return "California";
+  } if (school.includes("University of California")) {
+      usableWords[0] = "UC";
+      return usableWords.join("_");
+    }
+   if (school.includes("California State University")) {
+      if (
+        splitSchool[splitSchool.length - 1] == "Fresno" || 
+        splitSchool[splitSchool.length - 1] == "Sacramento" || 
+        splitSchool[splitSchool.length - 1] == "Long Beach"
+        ) {
+        return splitSchool[splitSchool.length - 1].concat("_State");
+      } else {
+        return "Cal_St_".concat(splitSchool[splitSchool.length - 1]);
+      }
+  }
+}
+
+const getNC = function (school) {
+    let splitSchool = school.split(" ");
+    if (school.includes("Chapel Hill")) {
```

Comment:
Just a heads up javascript has a ternary operator...can make this kind of thing more concise